src/main/io/gps.c

```diff
@@ -406,7 +416,6 @@ void gpsInit(void)
     gpsData.platformVersion = UBX_VERSION_UNDEF;
 
 #ifdef USE_DASHBOARD
-    gpsData.errors = 0;
```

Comment:
Was this a bug ?

Reply:
IMO only dashboard is using it, so more an inconsistency